Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.h

```diff
@@ -94,7 +94,7 @@ class RemoteLayerTreeHost {
     HashMap<WebCore::GraphicsLayer::PlatformLayerID, std::unique_ptr<RemoteLayerTreeNode>> m_nodes;
     HashMap<WebCore::GraphicsLayer::PlatformLayerID, RetainPtr<WKAnimationDelegate>> m_animationDelegates;
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
-    HashMap<WebCore::GraphicsLayer::PlatformLayerID, CGRect> m_overlayRegionsWithIDs;
+    HashMap<WebCore::GraphicsLayer::PlatformLayerID, const RemoteLayerTreeNode*> m_overlayRegionNodesWithIDs;
```

Comment:
Seems like a bad idea to store raw pointers here. Maybe a strong or weak reference?

Reply:
Oh, they're not refcounted...

Reply:
(can you prove that this is safe?)

Reply:
oh good point. I'll change this from a map to a set. The Map is storing <layerID, Node> but that is not needed, I can just store the layerID